proto/google/events/cloud/audit/v1/README.md

```diff
@@ -0,0 +1,25 @@
+## google.events.cloud.audit.v1
```

Comment:
Having these examples is great! What are the various ways we hope they get used? For example, might we want to generate reference documentation or link this from IDEs?

As we think about shell/protocol samples, I try to remember this is the primary reference for developers in every language for which we don't provide a library.

Reply:
I'm going to dig up a couple ideas of having a more common reference for curl samples. For now, having them somewhere is better than what we have today.

Talked to Lawrence about this today. Open to ideas!